### DISCUSSION TITLE
Can we have in the documentation some examples for example, login pages ,landing pages and etc, thanks.

### DISCUSSION THREAD

**Discussion by mariojgt** [saadeghi/daisyui#508] [Ideas / feature requests]
Just a suggestion i am not readly good with ui design buy i realy like this UI if is possible to have a part in the website where you have examples for login page or something similar to https://tailwindcomponents.com/ that would be cool.

**Comment by fanckush** [comment]
any updates on this?

**Reply by saadeghi** [reply]
I see this as a separate project. I'm planning to start working on it from next month but nothing is clear yet.

**Comment by alok-old** [comment]
Did daisyui website actually had those until recently? I remember using them (login page) but now it seems to be missing. I am currently in an existential crisis since something I used recently is now missing and I am scratching my head that if it even existed in the first place

**Reply by saadeghi** [reply]
You're probably talking about this example: https://daisyui.com/components/hero/#hero-with-form

**Reply by alok-old** [reply]
Yes that's it, thank you
